stock_orderpoint_manual_procurement/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+from datetime import datetime
+from openerp.addons import decimal_precision as dp
+from openerp.tools import float_compare, float_round
+
+
+UNIT = dp.get_precision('Product Unit of Measure')
+
+
+class StockWarehouseOrderpoint(models.Model):
+    _inherit = 'stock.warehouse.orderpoint'
+
+    @api.multi
+    @api.depends("product_min_qty", "product_id", "qty_multiple")
+    def _compute_procure_recommended(self):
+        procurement_model = self.env['procurement.order']
+        for op in self:
+            op.procure_recommended_date = \
+                procurement_model._get_orderpoint_date_planned(
+                    op, datetime.today())
+            procure_recommended_qty = 0.0
+            prods = procurement_model._product_virtual_get(op)
+            if prods is None:
+                continue
+            if float_compare(prods, op.product_min_qty,
+                             precision_rounding=op.product_uom.rounding) < 0:
+                qty = max(op.product_min_qty, op.product_max_qty) - prods
```

Comment:
@aheficent minimum qty is a trigger to let procurement engine start procuring. So if the stock level drop below minimum qty it will trigger procurement. Max qty will tell procurement engine how many will it procure. Here are few links regarding stock level:

https://www.lokad.com/min-max-inventory-planning-definition
http://accountlearning.blogspot.co.id/2010/05/minimum-level-or-safety-stock-and-its.html